utils/movable.py

```diff
@@ -0,0 +1,20 @@
+"""
+Common class for both snake and ladder.
+Given a source this can tell target.
+"""
+class Movable():
```

Comment:
This can be moved to the Board class itself as both snake and ladder are part of it.

Reply:
kept this is different class as because if a new entity comes in the board, then it can inherit this class and can do further modification.

Reply:
Can yo give any example of a new entity and how keeping it here will help?

Reply:
Consider that points are given for every move. But if one encounters a snake, points are deducted. Consider there are two types of snakes. One snake will reduce x points whereas the other snake will reduce 2x points. Similarly it can be extended to n different type of snakes.

Reply:
I agree that it should not be inside Board class. Ideally, Snake and Ladder should have inherited from it.

Also, "because if a new entity comes in the board" is not a good reasoning. Please google for "YAGNI"